### DISCUSSION TITLE
Hidden GBs on /nsm storage of Elasticsearch

### DISCUSSION THREAD

**Discussion by simoneBIP** [Security-Onion-Solutions/securityonion#7884] [Unsupported Versions] [answered]
Hi everybody, 
I set on my SIEM the storage limit for elasticsearch data to 600 GB during installation but there is an issue and the SIEM reaches 99% disk usage. 
If I run
`#du -sh /nsm/`

I see 592 GB but if I put "df -h" I see 1TB. How is it possible? 
result of: 
```
#df -h
Filesystem Size Used Avail Use% Mounted on 
/dev/sda2 497M 190M 308M 39% /boot 
/dev/mapper/system-tmp 2.0G 33M 2.0G 2% /tmp 
/dev/mapper/system-nsm 1007G 955G 52G 95% /nsm
```

[xxx_standalone.sls.txt](https://github.com/Security-Onion-Solutions/securityonion/files/8601512/xxx_standalone.sls.txt)

Can anyone help me to understand where is the problem and where are these hidden GBs of storage? 
Thanks in advance,
Simone

**Comment by trocade** [comment, marked as answer]
Hi,

Try with lsof and search for deleted. 
Most likely one docker instance  has deleted files but not succesfully  been released back to the OS.

To resolve: with the docker instance identified (in my case mainly filebeat) a so-restart filebeat will free the storage.

**Reply by simoneBIP** [reply]
Apparently as you suggested Filebeat was deleting so many file that have not been freed by the OS from the storage. 
Thank you very much for your support!
